mail/src/test/java/org/jboss/as/quickstarts/mail/TestUtils.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2024 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.as.quickstarts.mail;
+
+public class TestUtils {
+    static final String DEFAULT_SERVER_HOST = "http://localhost:8080/mail";
+    static final String API_PATH = "/home.jsf";
```

Comment:
I think this variable could be renamed to something more descriptive, for example `HOME_PATH` or `MAINPAGE_PATH`.

Reply:
refactored the code changes.